lib/ssl/src/tls_client_ticket_store.erl

```diff
@@ -157,7 +157,7 @@ code_change(_OldVsn, State, _Extra) ->
 
 
 -spec format_status(Opt :: normal | terminate,
-                    Status :: list()) -> Status :: term().
+                    ListStatus :: list()) -> Status :: term().
```

Comment:
maybe call it StatusData as that is what gen_server calls it and it is a gen_server callback!

Reply:
since the implementation simply returns the `Status`, I will instead change the returned type of `format_status` to be also a `list()`, instead of `term()`.